src/components/ui/ContextMenu/fragments/ContextMenuTrigger.tsx

```
@@ -0,0 +1,41 @@
+  className?: string;
+} & MenuPrimitiveProps.Trigger;
+
+const ContextMenuTrigger = ({ children, className, ...props }:ContextMenuTriggerProps) => {
+    
+    const context = React.useContext(ContextMenuContext);
+    if (!context) {
+        console.log('ContextMenuTrigger should be used in the ContextMenuRoot');
+        return null;
+    }
+    const { rootClass, setCoords, setIsOpen } = context;
+    const contextTriggerRef = React.useRef<HTMLSpanElement>(null);
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

Hooks should not be called after an early return.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)